### ISSUE TITLE
Anthropic (claude) support

### ISSUE THREAD

**Issue by omri-suissa-clearmash** [dmitry-brazhenko/SharpToken#32]
Can we use SharpToken for Anthropic? I could not find if claude is using "cl100k_base" or other encoding

**Comment by dmitry-brazhenko** [comment]
Hello @omri-suissa-clearmash !

Could you share a bit more what is claude or Anthropic?  What is there and how it works? 

Thanks

**Comment by omri-suissa-clearmash** [comment]
@dmitry-brazhenko 
claude is the LLM of Anthropic (https://www.anthropic.com/).
This is what I could find:
https://github.com/anthropics/anthropic-sdk-python/blob/e84645b07ca5267066700a104b4d8d6a8da1383d/src/anthropic/_tokenizers.py

**Comment by dmitry-brazhenko** [comment]
Thanks for sharing. 

I will check that. Probably they use some already existing encoding (cl100k_base) or just some custom one. I will check.

**Comment by omri-suissa-clearmash** [comment]
@dmitry-brazhenko also found this: https://github.com/19h/claude_tokenizer (rust)

**Comment by dmitry-brazhenko** [comment]
Hello @omri-suissa-clearmash !

I checked the algorithm. Seems that there is a difference, but it can be potentially implemented into Sharptoken lib. 
I will try to do that within a few days

**Comment by Mrgaton** [comment]
Any updates on this?

**Comment by alexander-bobrov-stealth** [comment]
Any updates on this?

**Comment by vwilson** [comment]
@alexander-bobrov-stealth I don't believe this repo is being maintained anymore - see the note in the current README.MD.  `Microsoft.ML.Tokenizers` is the way forward.

With that said, I have Opus working on this for fun and I will update my fork of this repo later on and give you a heads up.

**Comment by dmitry-brazhenko** [comment]
Hi!

Yes, we can add it here. If there is a PR, please feel free to raise it, we can merge it. 

Otherwise, I will have a look soon

**Comment by dmitry-brazhenko** [comment]
thanks a lot for doing this. I will have a look later today and will merge

**Comment by vwilson** [comment]
Happy to help. This project inspired me when I first discovered it.

**Comment by dmitry-brazhenko** [comment]
Merged the change, it should be available here: https://www.nuget.org/packages/SharpToken/2.0.5

Also add this info to readme

Thank you for contribution!
